magefiles/magefile.go

```diff
@@ -194,6 +193,9 @@ func Build() error {
 	if os.Getenv("MEMSTATS") == "true" {
 		buildTags = append(buildTags, "memstats")
 	}
+	if os.Getenv("MEMOIZE_BUILDERS") != "false" {
+		buildTags = append(buildTags, "memoize_builders")
```

Comment:
Because we know the lifecycle of coraza, notably lack of reloads, we can just always enable this here I think